### DISCUSSION TITLE
Reading/writing to worker filesystem from within a task?

### DISCUSSION THREAD

**Discussion by twitchyliquid64** [concourse/concourse#5674] [General]
In the common case (public CI for open source projects) this should not be possible, but very simple setups (ie: I have concourse setup on an intel NUC for pet things) it would be nice to be able to mark my task as privileged and be able to IO to the underlying filesystem.

Has anyone worked out how to do this?

I've tried the following in a privileged task:
1. Mount by device path
2. Mount by UUID
3. Mount devfs to mount by UUID

**Comment by vito** [comment]
I think this discussion covers the same thing: https://github.com/concourse/concourse/discussions/5495

tl;dr: There is currently no way to do this; it would require a code change, which I gave some details on in the linked thread.

**Comment by twitchyliquid64** [comment]
I eventually got something to work by using the `mock` resource to yeet out a shell script, which would ssh into itself to transfer a file. Had to turn on allow-access thou. Mad_hackz.jpg.

I've commented in the other thread, feel free to dupe this thread over to that one.
